tests/fv/test_multi_host.py

```diff
@@ -0,0 +1,62 @@
+from time import sleep
+import sh
+from sh import docker
+
+
+def multi_host():
+    docker_exec = docker.bake("exec")
+
+    host1_ip = docker.inspect("--format", "'{{ .NetworkSettings.IPAddress }}'", "host1").stdout.rstrip()
+    host2_ip = docker.inspect("--format", "'{{ .NetworkSettings.IPAddress }}'", "host2").stdout.rstrip()
+
+    calicoctl = "/code/dist/calicoctl"
+    etcd_port = "ETCD_AUTHORITY=%s:2379" % host1_ip
+    host1_listen = docker_exec.bake("-i", "host1", "bash", "-s")
+    host2_listen = docker_exec.bake("-i", "host2", "bash", "-s")
+
+    host1_listen(_in="%s %s reset || true" % (etcd_port, calicoctl))
+
+    host1_listen(_in="%s %s node --ip=%s" % (etcd_port, calicoctl, host1_ip))
+    host2_listen(_in="%s %s node --ip=%s" % (etcd_port, calicoctl, host2_ip))
+
+    calico_port = "DOCKER_HOST=localhost:2377"
```

Comment:
since you passing this in, I suspect the preferred way to do this is to specify docker -H=localhost:2377